systems/plants/@RigidBodyManipulator/contactConstraints.m

```diff
@@ -57,7 +57,8 @@
 % For now, all coefficients of friction are 1
 mu = ones(nC,1);
 
-d = obj.surfaceTangents(normal);
+d = surfaceTangentsmex(normal);
```

Comment:
You're right.

Is there a standard way to check for the presence of Eigen from MATLAB?  Would the existence of the compiled mex file be sufficient?